src/op_analytics/datapipeline/models/templates/erc20_contracts/erc20_contract_creation.sql.j2

```diff
@@ -0,0 +1,14 @@
+SELECT
+  ct.*
+  , 'erc20' AS token_standard
+  , sc.token_type
```

Comment:
Should we coalesce this to some other status, or keep it as null if not superchain erc20?

Reply:
i'm thinking of adding other `token_type` like NTT, OFT later after getting to understand the logic to fetch those better. But I assume most are just normal erc20 so maybe we should leave it as null?